src/components/ButtonPanel.js

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import Button from './Button';
+
+const ButtonPanel = () => (
+  <div id="button-panel">
+    <div>
+      <Button name="AC" />
+      <Button name="+/-" />
+      <Button name="%" />
+      <Button name="÷" />
+    </div>
+
+    <div>
+      <Button name="7" />
+      <Button name="8" />
+      <Button name="9" />
+      <Button name="x" />
+    </div>
```

Comment:
[Optional]

We could rewrite the ButtonPanel component to use two loops to make the `Button` instead of hardcoding them. By doing this, we keep our code [DRY](https://en.wikipedia.org/wiki/Don%27t_repeat_yourself).

We could achieve this by creating an array of symbol and loop through the array using [`map`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/map) to create the ButtonPanel component.